README.md

```diff
@@ -20,9 +20,56 @@ Via Composer
 $ composer require php-http/php-http-bundle
 ```
 
+Enable the bundle in your kernel:
+ 
+``` php
+<?php
+// app/AppKernel.php
+
+public function registerBundles()
+{
+    $bundles = array(
+        // ...
+        new Http\ClientBundle\HttpClientBundle(),
+    );
+}
+```
 
 ## Usage
 
+TODO: move this to php-http-documentation? or keep it here?
```

Comment:
Agreed. I tend to follow the following practice: usage section contains a small description about the package, the documentation section contains a link to the relevant documentation. Like [here](https://github.com/php-http/adapter#usage)